### PR TITLE
[xbmc/win32/improvement] Finally get rid of the VS solution!

### PR DESCRIPTION
**Pull request by Paxxi**
This drops the VS2010Express directory and modifies
BuildSetup.bat to only use cmake.

No more horrible xml merge conflicts :)

### CODE REVIEW

**Comment by Paxxi** [comment]
jenkins build this please

**Comment by Voyager1** [comment]
sorry if I missed this, but how would Windows devs then develop/debug the app?

**Comment by Paxxi** [comment]
@Voyager1 cmake generates a VS solution that behaves as usual, it has a bit different layout in the projects but other than that, it's the same workflow

**Comment by MartijnKaijser** [comment]
See
https://github.com/xbmc/xbmc/blob/master/project/cmake/README.md

**Comment by Voyager1** [comment]
thanks @Paxxi and @MartijnKaijser !

File: project/Win32BuildSetup/BuildSetup.bat [hunk 1]
```diff
@@ -56,61 +55,18 @@ FOR %%b in (%1, %2, %3, %4, %5, %6) DO (
   IF %%b==nomingwlibs SET buildmingwlibs=false
   IF %%b==nobinaryaddons SET buildbinaryaddons=false
   IF %%b==sh SET useshell=sh
-  IF %%b==cmake SET cmake=1
 )
 
 SET buildconfig=Release
-set WORKSPACE=%CD%\..\..
+set WORKSPACE=%CD%\..\..\kodi-build
```
Comment (wiromare) [inline project/Win32BuildSetup/BuildSetup.bat]: you forgot to cutoff some kodi-build

set EXE="%WORKSPACE%\kodi-build\%buildconfig%\%APP_NAME%.exe"
set PDB="%WORKSPACE%\kodi-build\%buildconfig%\%APP_NAME%.pdb"
set D3D="%WORKSPACE%\kodi-build\D3DCompile*.DLL"

Comment (Paxxi) [inline project/Win32BuildSetup/BuildSetup.bat]: Yeah I was on my way out the door so I just pushed, cleaning it up and doing it properly today

**Comment by Paxxi** [comment]
jenkins build this please

**Comment by ksooo** [comment]
Thumbs up for this. Especially as I do not have Windows and always had to beg some kind developer to update the solution files whenever I added new source files.

Now let's kill the Xcode project files.

**Pull request merged**